python/pyspark/ml/connect/functions.py

```diff
@@ -58,7 +58,7 @@ def _test() -> None:
         print("Not supported in no-GIL mode", file=sys.stderr)
         sys.exit(0)
 
-    from pyspark.testing import should_test_connect
+    from pyspark.testing.utils import should_test_connect
```

Comment:
Oh, this is not a test-only change because this is `python/pyspark/ml/connect/functions.py`, @zhengruifeng .